keras_core/backend/jax/random.py

```diff
@@ -3,16 +3,35 @@
 from keras_core.backend.config import floatx
 from keras_core.random.seed_generator import SeedGenerator
 from keras_core.random.seed_generator import draw_seed
-from keras_core.random.seed_generator import make_default_seed
 
 
 def jax_draw_seed(seed):
-    if isinstance(seed, jax.Array):
-        return seed
+    if isinstance(seed, SeedGenerator):
+        return seed.next()
     else:
         return draw_seed(seed)
 
 
+def make_default_seed():
+    return None, jax.random.PRNGKey(42)
```

Comment:
The default seed should not be fixed, it should be drawn from Python `random`

Reply:
I know. I kept if just for the sake of demonstration. Will replace it with a random selection